### PR TITLE
skypeforlinux: gnome2 cleanup

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Removed/updated gnome2 references (Part of #39976).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jtojnar** [comment]
Note that we will want to get rid of GConf from `gnome3` package set as well, it should not even be there.

**Comment by jtojnar** [comment]
Note that we will want to get rid of GConf from `gnome3` package set as well. It should not even be there, we should probably move it to a new package set called `abandonware`, if not remove it completely.

**Comment by worldofpeace** [comment]
Chromium uses gesettings only now. Soon electron will follow soon. But there are still lots of applications thtat need it so I think `abandonware` would be ok since it will soon see it's death :+1:

File: pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix [hunk 1]
```diff
@@ -22,17 +22,18 @@ let
     glibc
     libsecret
 
-    gnome2.GConf
-    gnome2.gdk_pixbuf
-    gnome2.gtk
-    gnome2.pango
+    gnome3.gconf
+    gnome3.gdk_pixbuf
```
Comment (jtojnar) [inline pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix]: Top-level `gdk_pixbuf` and `gtk2` should be used.

**Pull request merged**